test/SparqlAntlrParserTestHelpers.h

```diff
@@ -206,38 +207,39 @@ MATCHER_P(IsLiteral, value, "") {
 
 // _____________________________________________________________________________
 
-auto IsExpression = [](const std::string& descriptor) {
+auto IsExpression = [](const std::string& descriptor)
+    -> testing::Matcher<const sparqlExpression::SparqlExpressionPimpl&> {
```

Comment:
One thing in general: If you work on `std::string` etc, then you can just use `Eq`. `StrEq` in my understanding is just needed if you compare two `const char*`s to make it clear, that you don't want to compare the pointer values as integers.